### PR TITLE
feat: add scriptTimeout option

### PR DESCRIPTION
**Pull request by marcysutton**
Adds a configurable option to the Grunt-axe-webdriver plugin. 

Closes https://github.com/dequelabs/grunt-axe-webdriver/issues/35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +78,12 @@ Default value: `null`
 
 Which tags to filter violations on
 
+#### scriptTimeout
+Type: `Number`
+Default value: `null`
+
+Number value to tell Webdriver to wait before timing out
```
Comment (marcysutton) [inline README.md]: I wanted to point to the Selenium Webdriver javascript docs, but I can't for the life of me find the correct documentation page.

**Comment by marcysutton** [comment]
Ugh this build....I forgot to rebase from develop, and now some of the integration tests are failing since axe-core has been updated. Working on a fix.

File: README.md (hunk 1 above)
Comment (stephenmathieson) [reply README.md]: Perhaps mention that WebDriver will timeout an _injected script_ after this number of ms. It should be clear that this is not a page-load timeout.

**Comment by marcysutton** [comment]
@WilcoFiers this one is ready for another review. There were always failing axe-core rules, but they were under the threshold of 2 before. The new level-one-heading put it over the threshold, so I added a heading to make it pass.

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**